contracts/Delay.sol

```diff
@@ -78,8 +79,11 @@ contract Delay is AragonApp, IForwarder {
     * @param _delayedScriptId The ID of the script execution to pause
     */
     function pauseExecution(uint256 _delayedScriptId) external auth(PAUSE_EXECUTION_ROLE) {
+        DelayedScript storage delayedScript = delayedScripts[_delayedScriptId];
         require(!_isExecutionPaused(_delayedScriptId), ERROR_CAN_NOT_PAUSE);
-        delayedScripts[_delayedScriptId].pausedAt = getTimestamp64();
+        require(getTimestamp64() < delayedScript.executionTime, ERROR_SCRIPT_EXECUTION_PASSED);
```

Comment:
What do you think of storing the timestamp in a variable so we don't call the `getTimestamp64` function  twice and save some gas?

Reply:
An extra line of code for <50 gas probably isn't worth it. Personally, I prefer an "optimization threshold" of 1 SSTORE -- i.e. make code as simple as possible, and allow a little complexity if you can save around an SSTORE worth of gas.

Reply:
Sounds reasonable!